turbinia/task_manager.py

```diff
@@ -702,11 +780,14 @@ def _backend_setup(self, server=True, *args, **kwargs):
   def process_tasks(self):
     completed_tasks = []
     for task in self.tasks:
+      check_timeout = False
       psq_task = task.stub.get_task()
       # This handles tasks that have failed at the PSQ layer.
       if not psq_task:
+        check_timeout = True
```

Comment:
Same as above with Tasks not yet created/unknown state if they would have a start_time to check against?

Reply:
(answered above, but yes they should)